console/src/components/Services/RemoteSchema/Permissions/Container.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { ReactElement } from 'react';
+import CommonTabLayout from '../../../Common/Layout/CommonTabLayout/CommonTabLayout';
+import { NotFoundError } from '../../../Error/PageNotFound';
+import { appPrefix } from '../constants';
+import styles from '../RemoteSchema.scss';
+import { RSPContainerProps } from './types';
+
+const tabInfo = {
+  details: {
+    display_text: 'Details',
+  },
+  modify: {
+    display_text: 'Modify',
+  },
+  permissions: {
+    display_text: 'Permissions',
+  },
+};
+
+const RSPContainer: React.FC<RSPContainerProps> = ({
```

Comment:
Don't understand the name `Container`. `Container` (or `Page`) suggests that it will be connected to Redux, but it's just a component.